src/Libraries/Microsoft.Extensions.TimeProvider.Testing/FakeTimeProvider.cs

```diff
@@ -7,88 +7,125 @@
+    private DateTimeOffset _now = new(2000, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
+    private TimeZoneInfo _localTimeZone = TimeZoneInfo.Utc;
+    private int _wakeWaitersGate;
 
     /// <summary>
-    /// Gets the time which was used as the starting point for the clock in this <see cref="FakeTimeProvider"/>.
+    /// Initializes a new instance of the <see cref="FakeTimeProvider"/> class.
     /// </summary>
     /// <remarks>
-    /// This can be set by passing in a <see cref="DateTimeOffset"/> to the constructor
-    /// which takes the <c>epoch</c> argument. If the default constructor is used,
-    /// the clock's start time defaults to midnight January 1st 2000.
+    /// This creates a provider whose time is initially set to midnight January 1st 2000.
+    /// The provider is set to not automatically advance time each time it is read.
     /// </remarks>
     [Experimental]
-    public DateTimeOffset Epoch { get; } = new DateTimeOffset(2000, 1, 1, 0, 0, 0, 0, TimeSpan.Zero);
+    public FakeTimeProvider()
+    {
+        StartDateTime = _now;
+    }
 
     /// <summary>
     /// Initializes a new instance of the <see cref="FakeTimeProvider"/> class.
     /// </summary>
+    /// <param name="startDateTime">The initial time and date reported by the provider.</param>
     /// <remarks>
-    /// This creates a clock whose time is set to midnight January 1st 2000.
-    /// The clock is set to not automatically advance every time it is read.
+    /// The provider is set to not automatically advance time each time it is read.
     /// </remarks>
-    public FakeTimeProvider()
+    [Experimental]
+    public FakeTimeProvider(DateTimeOffset startDateTime)
     {
-        _localTimeZone = TimeZoneInfo.Utc;
-        _now = Epoch;
+        _ = Throw.IfLessThan(startDateTime.Ticks, 0);
+
+        _now = startDateTime;
+        StartDateTime = _now;
     }
 
     /// <summary>
     /// Initializes a new instance of the <see cref="FakeTimeProvider"/> class.
     /// </summary>
-    /// <param name="epoch">The starting point for the clock used by this <see cref="FakeTimeProvider"/>.</param>
+    /// <param name="startDateTime">The initial time and date reported by the provider.</param>
+    /// <param name="autoAdvanceAmount">The amount of time by which to automatically advance time whenever it is read.</param>
     [Experimental]
-    public FakeTimeProvider(DateTimeOffset epoch)
-        : this()
+    public FakeTimeProvider(DateTimeOffset startDateTime, TimeSpan autoAdvanceAmount)
     {
-        Epoch = epoch;
-        _now = epoch;
+        _ = Throw.IfLessThan(startDateTime.Ticks, 0);
+        _ = Throw.IfLessThan(autoAdvanceAmount.Ticks, 0);
+
+        _now = startDateTime;
+        _autoAdvanceAmount = autoAdvanceAmount;
+        StartDateTime = _now;
     }
 
+    /// <summary>
+    /// Gets the starting date and time for this provider.
+    /// </summary>
+    [Experimental]
+    public DateTimeOffset StartDateTime { get; }
```

Comment:
Isn't it redundant to include the word "DateTime" in the property name?

It's more succinct to call `Start`, or as discussed at length in the previous PR, `Epoch`.

Reply:
Done.